### DISCUSSION TITLE
Generation of webhooks notifications

### DISCUSSION THREAD

**Discussion by susangz** [SecurityBrewery/catalyst#355] [Ideas]
Hi! 

Catalyst looks a really great open-source SOAR solution. Do you foresee to generate webhooks notifications of the actions taken by the users on the system (e.g. changes in the tickets or in the tasks) so they can be integrated with other tools/scripts?  I mean something similar to the webhooks feature provided by TheHive (https://docs.thehive-project.org/thehive/legacy/thehive3/admin/webhooks/ or https://blog.agood.cloud/posts/2022/07/02/thehive5-webhooks/) 

Thanks!

Best regards, 
    Susana

**Comment by cugu** [comment]
Hi Susana, 

Yes, webhooks are a great mechanic and would fit catalyst very well. They are on the list to implement, but I cannot make any promises for a timeframe for this feature.

**Comment by cugu** [comment]
This is implemented in [v0.12.0](https://github.com/SecurityBrewery/catalyst/releases/tag/v0.12.0)
